### PR TITLE
Fix PAS Polling

### PR DESCRIPTION
**Pull request by blangley28**
Polling time was updated in the latest version of the IG so the polling is updated to reflect the new specification. Along the way I noticed polling was not done correctly and updated that with both polling and ws subscriptions.

There's also a ton of prettier updates - not sure why but I think that's fine

### CODE REVIEW

File: src/components/PriorAuth/PriorAuth.jsx [hunk 1]
```diff
@@ -46,46 +46,39 @@ export default class PriorAuth extends Component {
...
-   * Specification: no more than 4 times in first hour
+   * Specification: every 5 min for the first half hour
    *                no more than once per hour after that
    *                at least once every 12 hours
    */
   polling() {
-    // let hourInSec = 3600;
-    let hourInSec = 32; // For testing purposes make an hour 16 seconds
+    // const hourInSec = 3600;
+    const hourInSec = 32; // For testing purposes make an hour 16 seconds
```
Comment (garypgryan) [inline src/components/PriorAuth/PriorAuth.jsx:61]: Is there a way to invoke the testing version via a config file or command line parameter?   I was a bit confused the first time I read this.

File: src/components/PriorAuth/PriorAuth.jsx [hunk 2]
```diff
@@ -131,7 +124,7 @@ export default class PriorAuth extends Component {
   handleGetLink() {
     console.log(window.location.href);
     this.setState({
-      showLink: true
+      showLink: true,
```
Comment (garypgryan) [inline src/components/PriorAuth/PriorAuth.jsx:127]: Why is there a trailing comma?   (I'm not a REACT programmer, so I found this confusing.

**Review by garypgryan — APPROVED** [review]
Looks good; I had a few informational questions.

File: src/components/PriorAuth/PriorAuth.jsx (hunk 1 above)
Comment (blangley28) [reply src/components/PriorAuth/PriorAuth.jsx]: Added a config file

File: src/components/PriorAuth/PriorAuth.jsx (hunk 2 above)
Comment (blangley28) [reply src/components/PriorAuth/PriorAuth.jsx:126]: These were all automatically added by Prettier in vscode. The Prettier plugin auto formats based on the project files so it must be defined somewhere there to make this a rule

**Pull request merged**